### PR TITLE
fix: show fingerprint should not be sticky [WPB-7117]

### PR DESCRIPTION
**Pull request by theswiftway**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-7117" title="WPB-7117" target="_blank">BugWPB-7117</a>  View must not be sticky
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Show my fingerprint is sticky. It should be non-sticky

### Solutions

Show my fingerprint is non-sticky

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [x] I have added automated test to this contribution

#### How to Test

Go to Conversations -> Tap on Header -> Tap on a user -> Tap on Devices ->  Tap on a client
Expectation: Show my device fingerprint is not sticky

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by KaterinaWire — DISMISSED** [review]
We shouldn't have "Show my device fingerprint" for self user devices. Please check Figma.

**Review by caldrian — APPROVED** [review]
nice work!

File: wire-ios/Wire-iOS Tests/ReferenceImages/ProfileDeviceDetailsViewTests/testWhenE2eidentityIsEnabledAndCertificateIsExpiredForOtherClient.1.png
Comment (caldrian) [inline wire-ios/Wire-iOS Tests/ReferenceImages/ProfileDeviceDetailsViewTests/testWhenE2eidentityIsEnabledAndCertificateIsExpiredForOtherClient.1.png:1]: Some snapshot images a cropped.

Comment (theswiftway) [reply wire-ios/Wire-iOS Tests/ReferenceImages/ProfileDeviceDetailsViewTests/testWhenE2eidentityIsEnabledAndCertificateIsExpiredForOtherClient.1.png:1]: This was not needed. Deleted it. If the layout is longer than device size, it doesn't take the full snapshot

**Review by David-Henner — APPROVED** [review]

**Pull request merged**